google/auth/transport/grpc.py

```diff
@@ -114,11 +124,91 @@ def secure_authorized_channel(
 
         # Create a channel.
         channel = google.auth.transport.grpc.secure_authorized_channel(
-            credentials, 'speech.googleapis.com:443', request)
+            credentials, regular_endpoint, request,
+            ssl_credentials=grpc.ssl_channel_credentials())
 
         # Use the channel to create a stub.
         cloud_speech.create_Speech_stub(channel)
 
+        # There are actually a couple of options to create a channel, depending
```

Comment:
This looks great! Thank you for writing this. 

Could you move this out of the existing examples section and format it more like [this docstring](https://github.com/googleapis/google-auth-library-python/blob/master/google/auth/impersonated_credentials.py#L114-L173)? The generated [html docs](https://googleapis.dev/python/google-auth/latest/reference/google.auth.impersonated_credentials.html) are easier to read.

You can view the docs locally by running `nox -s docs` and opening the directory in the browser.